tensorboard/main.py

```diff
@@ -63,6 +64,9 @@ def run_main():
     raise AssertionError("absl.app.run() shouldn't return")
   except ImportError:
     pass
+  except base_plugin.FlagsException as e:
+    print("Error: " + str(e))
```

Comment:
Should probably be printed to stderr? Consider

```py
from tensorboard.util import tb_logging

# ...
except base_plugin.FlagsException as e:
  logger = tb_logging.getLogger()
  logger.error("%s", e)
```

Reply:
tb.logging prefixes the message with some goobledygook:

`E0208 21:29:29.716873 140735652410240 main.py:70]`

I've changed to using stderr with print.